iam-test-client/docker/Dockerfile.prod

```diff
@@ -1,10 +1,10 @@
-FROM openjdk:8
+FROM openjdk:11
```

Comment:
I think this has been missed since the upgrade to Java 17, I've been having compatibility issues. I suggest changing this to
```
FOM openjdk:17
```